doc/reference/reference_lua/box_index/unique.rst

```diff
@@ -0,0 +1,15 @@
+.. _box_index-unique:
+
+===============================================================================
+index_object:unique
```

Comment:
This should have a dot: `index_object.unique`

Reply:
fixed in 22a984c